<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridMultiplierDialog</class>
 <widget class="QDialog" name="GridMultiplierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>557</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>grid multiplier</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <property name="spacing">
    <number>16</number>
   </property>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>setup grid multiplier</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnminimumwidth="200,150,0,0">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>initial position (x,y)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="initialPosXEdit">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="initialPosYEdit">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="velXCheckBox">
        <property name="text">
         <string>set velocity x</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="velYCheckBox">
        <property name="text">
         <string>set velocity y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="initialVelXEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="initialVelYEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="angleCheckBox">
        <property name="text">
         <string>set angle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="initialAngleEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="angVelCheckBox">
        <property name="text">
         <string>set angular velocity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="initialAngVelEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>horizontal number</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="horizontalNumberEdit">
        <property name="text">
         <string>4</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>interval</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="horIntervalEdit">
        <property name="text">
         <string>50.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>velocity increment (x,y)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="horVelXIncEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="horVelYIncEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>angle increment</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="horAngleIncEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>angular velocity increment</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLineEdit" name="horAngVelIncEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>vertical number</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLineEdit" name="verticalNumberEdit">
        <property name="text">
         <string>4</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>interval</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLineEdit" name="verIntervalEdit">
        <property name="text">
         <string>50.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>velocity increment (x,y)</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLineEdit" name="verVelXIncEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="3">
       <widget class="QLineEdit" name="verVelYIncEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>angle increment</string>
        </property>
       </widget>
      </item>
      <item row="14" column="2">
       <widget class="QLineEdit" name="verAngleIncEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>angular velocity increment</string>
        </property>
       </widget>
      </item>
      <item row="15" column="2">
       <widget class="QLineEdit" name="verAngVelIncEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>initialPosXEdit</tabstop>
  <tabstop>initialPosYEdit</tabstop>
  <tabstop>velXCheckBox</tabstop>
  <tabstop>velYCheckBox</tabstop>
  <tabstop>initialVelXEdit</tabstop>
  <tabstop>initialVelYEdit</tabstop>
  <tabstop>angleCheckBox</tabstop>
  <tabstop>initialAngleEdit</tabstop>
  <tabstop>angVelCheckBox</tabstop>
  <tabstop>initialAngVelEdit</tabstop>
  <tabstop>horizontalNumberEdit</tabstop>
  <tabstop>horIntervalEdit</tabstop>
  <tabstop>horVelXIncEdit</tabstop>
  <tabstop>horVelYIncEdit</tabstop>
  <tabstop>horAngleIncEdit</tabstop>
  <tabstop>horAngVelIncEdit</tabstop>
  <tabstop>verticalNumberEdit</tabstop>
  <tabstop>verIntervalEdit</tabstop>
  <tabstop>verVelXIncEdit</tabstop>
  <tabstop>verVelYIncEdit</tabstop>
  <tabstop>verAngleIncEdit</tabstop>
  <tabstop>verAngVelIncEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GridMultiplierDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velXCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>initialVelXEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velXCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horVelXIncEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velXCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>verVelXIncEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velYCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>initialVelYEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velYCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horVelYIncEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velYCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>verVelYIncEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angVelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>initialAngVelEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angVelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horAngVelIncEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angVelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>verAngVelIncEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>initialAngleEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horAngleIncEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>verAngleIncEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
